packages/cloud-functions/src/firebase.ts

```diff
@@ -31,3 +31,12 @@ export function saveTxHashProvider(address: string, txHash: string, provider: st
       )
     )
 }
+
+export async function fetchFromFirebase(path: string) {
+  const snapshot = await database().ref(path).once('value')
+  return snapshot.val()
+}
+
+export async function updateFirebase(path: string, value: any) {
+  await database().ref(path).set(value)
```

Comment:
The function is called `update` but it's actually using `set` which do different things (`update` doesn't delete fields you don't send, `set` does). Lets change one of them to make it clearer :)

Reply:
lol, I'm sorry :(